minorminer/utils/parallel_embeddings.py

```diff
@@ -197,20 +240,113 @@ def find_multiple_embeddings(
     return embs
 
 
+def lattice_size(T: Optional[nx.Graph] = None) -> int:
```

Comment:
`T` doesn't look optional, but required.
```suggestion
def lattice_size(T: nx.Graph) -> int:
```